developer/decision-records/2023-09-06-identity-trust/README.md

```diff
@@ -0,0 +1,105 @@
+# Technical Direction for a Robust Decentralized EDC Identity and Trust System
+
+## Decision
+
+This decision record establishes the technical direction for developing an EDC identity and trust system that can be
+used in
+place of the existing OAuth2-based system when required. This system will be built on existing EDC infrastructure,
+including the `Identity Hub.`
+
+## Rationale
+
+Identity and trust are the cornerstones of a dataspace. Currently, the EDC has production-quality support for the
+exchange of identity and claims information using
+the [OAuth 2 Client Credentials Grant](https://datatracker.ietf.org/doc/html/rfc6749#section-4.4). For many use cases,
+relying on a centralized OAuth 2 Authorization Server is sufficient. However, there are a class of uses-cases which
+require a decentralized approach to exchanging identity and claims data. For example, operational continuity
+requirements may determine that a centralized Authentication Server poses too great a risk of service disruption. Or,
+privacy rules may dictate that claims verification must be done without requiring both the verifying and verified party
+being
+known to a third-party identity provider.
+
+The decentralized identity and trust system will enable dataspace participants to securely communicate and validate
+credentials using
+the [Dataspace Protocol Specifications](https://github.com/International-Data-Spaces-Association/ids-specification) (
+DSP) in a way that preserves privacy and limits the possibility of network disruption.
+
+## Approach
+
+The EDC identity and trust system will implement a set of message interactions (termed "protocols") for the following:
+
+- **Self-Issued Identity Tokens**: Obtaining and cryptographically validating self-issued tokens that contain identity
+  claims
+- **Verifiable Credential Presentation**: Communicating Verifiable Credentials (VC), Verifiable Presentations (VP), and
+  the cryptographic material needed to validate them
+- **Verifiable Credential Issuance**: Issuing and renewing Verifiable Credentials and providing a mechanism for
+  communicating credential revocation
+
+For convenience (and because not all stakeholders are EDC committers), the protocol specifications will be temporarily
```

Comment:
```suggestion
### Overview of Protocol Specifications
The protocol specifications will be temporarily
```

Reply:
The way I read this change is it is changing the title of the section to be an overview, which it's not. I'm fine to strike the line "For convenience" but not to change the section title.

Reply:
If you don't want to include this section title, you can revert the previous commits as they changed the heading level based on this...

Reply:
Reworded as per comment below.